### PR TITLE
lablgtk test fix

### PR DESCRIPTION
**Pull request by tschoonj**
hopefully addresses Homebrew/homebrew#40858

### CODE REVIEW

**Comment by bfontaine** [comment]
The test fails:

```
==> ocamlc -I +lablgtk2 lablgtk.cma gtkInit.cmo test.ml -o test
File "test.ml", line 2, characters 2-19:
Error: Unbound module GtkMain
Error: lablgtk: failed
```

File: Library/Formula/lablgtk.rb [hunk 1]
```diff
@@ -33,7 +33,46 @@ def install
...
+    cairo = Formula["cairo"]
+    fontconfig = Formula["fontconfig"]
+    freetype = Formula["freetype"]
+    gdk_pixbuf = Formula["gdk-pixbuf"]
+    gettext = Formula["gettext"]
+    glib = Formula["glib"]
+    gtkx = Formula["gtk+"]
+    libpng = Formula["libpng"]
+    pango = Formula["pango"]
+    pixman = Formula["pixman"]
```
Comment (xu-cheng) [inline Library/Formula/lablgtk.rb]: It doesn't seem like that any of these variables are being used.

File: Library/Formula/lablgtk.rb [hunk 2]
```diff
@@ -33,7 +33,46 @@ def install
         GtkMain.Main.init ()
       let _ = main ()
     EOS
-    system "ocamlc", "-I", "#{Formula["lablgtk"].opt_lib}/ocaml/lablgtk2", "lablgtk.cma", "gtkInit.cmo", "test.ml", "-o", "test"
+    ENV.libxml2
```
Comment (xu-cheng) [inline Library/Formula/lablgtk.rb]: This is used to alter `cflags` which isn't used in the test at all.

**Comment by tschoonj** [comment]
Thanks for that. Hadn't cleaned up properly after all this trial and error.

Looks good now. Feel free to pull it in at your convenience.

**Comment by xu-cheng** [comment]
Thanks @tschoonj

**Pull request closed without merging**